What's an HTTP/3 connection? (#4212)
quicwg/base-drafts: draft-ietf-quic-http.md

  frames."  References without this preface refer to frames defined in
  {{frames}}.

HTTP/3 connection:

> : A QUIC connection where the negotiated application protocol is HTTP/3.

Reviewer: ```suggestion
: A QUIC connection where the selected application protocol is HTTP/3.
```

make it more consistent with the notion of H3 selection in #4211

Reviewer 2: Huh, I always thought negotiated was preferred since we're using ALPN?

Reviewer: The spec also says that other methods of selecting H3 are possible. And if we're speaking generically, then that might not be a negotiation. But I'll let Mike make the editorial decision here.

Reviewer 3: Either works; but as ALPN involves a server selection and the N stands for negotiation, I don't think it matters.

Author: I'm inclined toward keeping negotiation here.

When HTTP fields and data are sent over QUIC, the QUIC layer handles most of
the stream management.  HTTP does not need to do any separate multiplexing when
using QUIC - data sent over a QUIC stream always maps to a particular HTTP

> transaction or to the HTTP/3 connection context as a whole.

Reviewer 4: suggestion:
```suggestion
transaction or to the entire HTTP/3 connection context.
```

In the client to server direction, the GOAWAY frame carries a Push ID encoded as
a variable-length integer.

> The GOAWAY frame applies to the connection as a whole, not a specific stream.  A

Reviewer 4: ```suggestion
The GOAWAY frame applies to the entire connection, not a specific stream.  A
```
